marimbabot_vision/scripts/detect.py

```diff
@@ -0,0 +1,58 @@
+import torch
+from PIL import Image
+from transformers import (DonutProcessor, VisionEncoderDecoderConfig,
+                          VisionEncoderDecoderModel)
+
+INPUT_FILE = './demo.png'
+MODEL = './model'
+
+# Load base model
+ved_config = VisionEncoderDecoderConfig.from_pretrained(MODEL)
+pre_processor = DonutProcessor.from_pretrained(MODEL)
+model = VisionEncoderDecoderModel.from_pretrained(MODEL, ignore_mismatched_sizes=True, config=ved_config)
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+model.to(device)
+
+# Load image
+image = Image.open(INPUT_FILE).convert('RGB')
+
+# Rotate image if needed
+if image.size[0] > image.size[1]:
+    image = image.transpose(Image.Transpose.ROTATE_90)
```

Comment:
we should figure out a way how to overcome this later because we may not know whether to rotate right or left

Reply:
This is mainly because lilypond generates them in portrait and it writes them directly to disk and I don't want to load and safe them again. And the script from Juli makes them as landscape, which is also reasonable. Currently we only have these two options, so the code works, but it is a bit ugly, I agree with that.

Reply:
We can open an issue/ticket for that